about.html

```diff
@@ -0,0 +1,150 @@
+        <section class="head-part" id="head">
+          <h1 class="mobile">
+            About Africa<br/>Tech
+            Week
+          </h1>
+          <h2 class="head-h2 desktop">
+            About Africa <br/>Tech
+            Week
+          </h2>
+          <p class="head-part-text">
+            The expert panels will cover public and private policy aspects and their implications advantages and opportunities. It will also include industry’s leaders from the private sector, civil society, investors, partners, experts, academics all with their expert insight into the different ways artificial intelligence,cryptocurrency,IoT, will affect the future of African Development.
+          </p>
+          <h4 class="head-h4">LET'S CONNECT</h4>
+          <h4 class="head-h4" ><strong>25-29 OCTOBER 2022</strong></h4>
+          <h4 class="head-h4">Virtual Event</h4>
+        </section>
+      </div>
+
+      <section id="stuff" class="text-center">
+        <h4>Afri Tech Week Logo 2022</h4>
+        <p>The main sessions will feature addresses from world authorities on aspects of the 4th Industrial Revolution (Industry 4.0). They will outline the challenges and different scenarios as well as the mind blowing economic opportunities Industry 4.0 can deliver for those countries and organisations that make preparations for it.</p>
+        <img src="./images/afri-tech-img.png" alt="">
+      </section>
+      <!-- Card section Mobile & Desktop -->
+      <section id="about-program">
+        <h4 class="text-center">See the past Afri Tech Week Events</h4>
+        <p class="text-center my-3 text-dark">The breakaways and focus sessions will include panels of world-leading authorities on the 4th Industrial Revolution’s effects on individual industry sectors, specific fields of production and different areas of commerce.</p>
+        <div class="container pb-3">
+          <div class="row justify-content-md-center prog-row">
+            <div class="col-md-6 prog-item">
+              <div class="card card-item">
+                <img class="prog-img" src="./images/prog-img.jpeg" alt="Card Background">
+                <div class="overlay">
+                  <img src="./images/harmburger_img.png" alt="">
+                  <h5>Lecture</h5>
+                  <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aperiam, aut?</p>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-6 prog-item">
+              <div class="card card-item">
+                <img class="prog-img" src="./images/prog-img.jpeg" alt="Card Background">
+                <div class="overlay">
+                  <img src="./images/harmburger_img.png" alt="">
+                  <h5>Lecture</h5>
+                  <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aperiam, aut?</p>
+                </div>
+              </div>
+            </div>
+            </div>
+          </div>
+        </div>
+      </section>
```

Comment:
- [ ] Kindly make sure this section design much correctly the one from the Behance template desig i.e. it should have acorrect image on the backgroung to stick to the expexpected design.

| Template design | Your Design |
| ---------------------- | --------------------- |
| ![Screenshot from 2022-09-17 00-45-52](https://user-images.githubusercontent.com/83042742/190827448-198510bc-bd3d-463b-9c85-e9034d3b59be.png) | ![Screenshot from 2022-09-17 00-43-07](https://user-images.githubusercontent.com/83042742/190827470-4be52c1b-89b4-49cb-85d2-e0793db18ee9.png) |